tasks/download_kubeconfig.yml

```diff
@@ -0,0 +1,21 @@
+---
+
+- name: Download k3s kubeconfig to localhost
+  ansible.builtin.fetch:
+    src: /etc/rancher/k3s/k3s.yaml
+    dest: "{{ k3s_download_kubeconf_path }}/{{ k3s_download_kubeconf_file_name }}"
+    flat: yes
+  delegate_to: "{{ k3s_control_delegate }}"
+  run_once: true
+  when:
+  - k3s_download_kubeconf | bool
+
+- name: Replace loopback IP by master server IP
+  ansible.builtin.replace:
+    path: "{{ k3s_download_kubeconf_path }}/{{ k3s_download_kubeconf_file_name }}"
+    regexp: '127.0.0.1'
+    replace: "{{ k3s_registration_address | default(hostvars[k3s_control_delegate].ansible_host) }}"
+  delegate_to: localhost
+  become: false
+  when:
```

Comment:
:pick: Move to `include_tasks` in `main.yml`

Reply:
Moved all that I can